File: NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/dto/request/FollowRequest.java
package io.twogether.nbe_5_7_2_02team.member.dto.request;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
public class FollowRequest {

    private Long followerId;
    private Long followingId;

    public static FollowRequest of(Long followerId, Long followingId) {
        return new FollowRequest(followerId, followingId);
    }
}

File: NBE_5_7_2_02TEAM/src/main/java/io/twogether/nbe_5_7_2_02team/member/dto/request/FollowRequest.kt
package io.twogether.nbe_5_7_2_02team.member.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotNull

@Schema(description = "팔로우 요청 DTO")
data class FollowRequest(
    @field:NotNull(message = "잘못된 팔로우 요청입니다.")
    @field:Schema(description = "팔로우 하는 회원 ID", example = "1")
    val followerId: Long,

    @field:NotNull(message = "잘못된 팔로우 요청입니다.")
    @field:Schema(description = "팔로우 받는 회원 ID", example = "2")
    val followingId: Long,
)
